src/common/colorspaces.c

```diff
@@ -945,7 +963,8 @@ static const char *_profile_names[] =
   N_("standard color matrix"),
   N_("enhanced color matrix"),
   N_("vendor color matrix"),
-  N_("alternate color matrix")
+  N_("alternate color matrix"),
+  N_("Prophoto RGB"),
```

Comment:
Bear with me, I'm no expert. But is it ok to have a Prophoto RGB color space into the list of matrix? I mean we don't have Adobe RGB here for example.

Reply:
That's an error, it shouldn't be there, I will fix that.